src/go/k8s/api/vectorized/v1alpha1/cluster_types.go

```diff
@@ -1215,27 +1220,28 @@ func (k KafkaAPI) IsMutualTLSEnabled() bool {
 // Admin API
 
 // GetPort returns API port
-func (a AdminAPI) GetPort() int {
+func (a AdminAPI) GetPort() int { //nolint:gocritic // struct will be still quite small
```

Comment:
Is there any reason to not make these pointer receivers instead of ignoring the linter?

Reply:
Yeah, that should be better. The linter made me think there were just a couple of them at the beginning